community-stabilization-fund-api/src/modules/checklists/components/ItemChecklistByBag.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 
 import type { FormResponse } from '../../../db';
 
+import { BagLabels } from './BagLabels';
 import { formResponseMock } from '../../../mocks';
 import {
   mapFormResponseToBagItems,
   mapFormResponseToRecipientInfo,
 } from '../utils';
 
-import { BagLabels } from './BagLabels';
```

Comment:
Generally we want to be able to see the components being rendered separately from utils, mocks, hooks, etc for the ease of debugging. I gotta update the eslint rules to respect the difference but putting this here to note it.